app/models/timdex_wrapper.rb

```diff
@@ -0,0 +1,57 @@
+class TimdexWrapper
+  def initialize
+    @timdex_http = HTTP.persistent(timdex_url)
+                       .headers(accept: 'application/json',
+                                'Accept-Encoding': 'gzip, deflate, br',
+                                'Content-Type': 'application/json',
+                                Origin: 'https://lib.mit.edu')
+  end
+
+  # Run a search
+  # @param term [string] The string we are searching for
+  # @return [Hash] A Hash with search metadata and an Array of {Result}s
+  def search(term)
+    @results = @timdex_http.timeout(http_timeout)
+                           .get(search_url(term))
+    JSON.parse(@results.to_s)
+  rescue StandardError => e
+    error = {}
+    error['error'] = e.to_s
+    error
+  end
+
+  private
+
+  def clean_term(term)
+    term.gsub('"', '\'')
+  end
+
+  # https://github.com/httprb/http/wiki/Timeouts
+  def http_timeout
+    if ENV['TIMDEX_TIMEOUT'].present?
+      ENV['TIMDEX_TIMEOUT'].to_f
+    else
+      6
+    end
+  end
+
+  def query(term)
```

Comment:
We had split out this functionality into a TimdexQueryBuilder class. I know that seems silly based on the current needs, but since we know that the QueryBuilder will eventually get quite complex I'd prefer to add the abstraction early and build on it rather than have to move this later (which we may keep putting off until we have a mega "search" model.